### ISSUE TITLE
Does not scroll to bottom of images loaded with ng-src on iOS

### ISSUE THREAD

**Issue by kmckee** [Luegg/angularjs-scroll-glue#39]
It looks like on Safari for iOS it's not handling images properly.  When a new image appears in a container that's glued to the bottom it only scrolls down a little bit, rather than scrolling down far enough to be able to see the entire image.  It doesn't seem to happen on android.

Here's a stripped down version of the code where I'm seeing the problem.
```
<div scroll-glue="true">
    <div ng-repeat="message in messages" class="message">
        <div>
            <div ng-if="message.type == 'image'" ng-cloak><img ng-src="{{ message.content }}"></div>
        </div>
    </div>
</div>
```

I also found an issue elsewhere that might be related:  https://github.com/matthewcodes/ping/issues/4

**Comment by kmckee** [comment]
I was able to work around this by changing the flag I'm using to enable scroll-glue to true, inside a timeout, after I've added the new image.
```
var updateMessages = function () {
    return $scope.thread.messages().then(function (messages) {
        $scope.messages = messages;
        $timeout(function() {
            $scope.enableScrollGlue = true;
        });
    });
};
```
A bit of a hack, and to be honest I don't totally understand what's going on....but figured I'd share in case anyone else was having a similar problem.

**Comment by LTroya** [comment]
@kmckee did it work with ionic? I tried it, appended a message to a view each 10 seconds, but the glue didn't work. In a web angular project, it works perfectly.

**Comment by sh-guti** [comment]
Didn't work for me also on ionic with ios

**Comment by oliversalzburg** [comment]
Just FYI, the way you're using `scroll-glue` in the original post, is not how it works (anymore).

You have to either simply use `scroll-glue` or provide an expression that is evaluated on the scope as the value for the `scroll-glue` attribute. When you set `scroll-glue="true"`, `true` is treated as an expression that is watched. This means that it expects to find a variable named "true" on the scope, which is not what you want.

Long story short, just set `scroll-glue` without a value.

**Comment by mwithington** [comment]
So this isn't working for me currently I'm experiencing the same issue.
